src/utils/theme.js

```diff
@@ -0,0 +1,500 @@
+import { createMuiTheme } from '@material-ui/core/styles';

```

Comment:
A comment with a link from where this was copied would be helpful for posterity.

Reply:
Applied Changes!